project/project/plugins.sbt

```diff
@@ -1 +0,0 @@
-addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.0-RC11")
```

Comment:
Why removed? This is not redundant setting.

Reply:
Exactly. I restored it in d17aed23576f9d9cec49e476b57956f4c045c580.